html/src/vr.js

```diff
@@ -79,7 +79,7 @@ CefSharp.BindObjectAsync(
         },
         layout: 'topCenter',
         theme: 'relax',
-        timeout: 6000
+        timeout: 3000
```

Comment:
👀

Reply:
lol yea I guess this is a bit of an opinionated change, maybe this should be a setting as well, more settings! 😆